### PR TITLE
Jacob distancebetween

### PR DESCRIPTION
**Pull request by cuthbert1221**
Adding the CSV Writer Module
This branch still has the unmerged changes from adding the tests for the geodist.py 
I can move these out if required.

### CODE REVIEW

**Review by ashketchupppp — CHANGES_REQUESTED** [review]
What you have here is exactly what I was hoping for, well done.

Perhaps we could merge this module with the csvreader module to make one "fileutil" module, seems a better idea than splitting them up. Not sure why we split them up in the first place.

Like I said in the slack message, the csvwriter test is testing two things: correct file reading and correct conversion of csv formatted strings to a list of dictionaries. Implicitly testing two things in one test is bad practice because if the test fails then you don't know which thing caused the test failure. Could you write a file writer function (make a new file for it and put it in the new fileutil module) and then use that to write your file? That way you can test that part and you are 100% sure it works when you use it in csvwriter. 

Approving with the requested changes, good work

**Comment by cuthbert1221** [comment]
Yes, I'll add the file writer function over the weekend.

One question, when you say "Approving with the requested changes"
Does this mean merging the current request and then add changes in the new one? Or wait until amendments then merge all together,

**Pull request closed without merging**
